Source/AbstractInterpretation/AbstractInterpretation.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <AssemblyName>Boogie.AbsInt</AssemblyName>
+    <RootNamespace>AbsInt</RootNamespace>
```

Comment:
Are these two still `AbsInt` because it would require internal changes to a bunch of source files to update them?